src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java

```diff
@@ -6,10 +6,14 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.imports.avoidstarimport;
+
 // xdoc section -- start
 import java.util.Scanner;         // OK
```

Comment:
@romani I thought we were removing OKs?

Reply:
Yes, I missed this.
@TanayMorakhia , please remove ok comments that has no text in comment.

Reply:
@romani Do we have OK removal check on xdoc examples? Is this just missing a suppression removal?

Reply:
done